Blog-projects/Widget-window-in-Qt-Quick-app/widgetFormHandler.cpp

```diff
@@ -0,0 +1,41 @@
+#include "widgetFormHandler.h"
+
+#include "fontcontrolswidgetsform.h"
+
+WidgetFormHandler::WidgetFormHandler(QObject *parent)
+    : QObject{parent}
+{
+    m_window = new FontControlsWidgetsForm();
+    m_window->setVisible(false);
+
+    connect(m_window, &FontControlsWidgetsForm::buttonClicked, this, &WidgetFormHandler::toggleWidgetsWindow);
+    connect(m_window, &FontControlsWidgetsForm::textChanged, this, &WidgetFormHandler::textChanged);
+    connect(m_window, &FontControlsWidgetsForm::fontChanged, this, &WidgetFormHandler::fontChanged);
+}
+
+const bool WidgetFormHandler::isVisible()
+{
+    return m_window->isVisible();
+}
+
+void WidgetFormHandler::setVisible(bool visible)
+{
+    m_window->setVisible(visible);
+    emit visibleChanged();
+}
+
+const QString WidgetFormHandler::text()
```

Comment:
the method is const, not the QString => move the const to the end

Reply:
Fixed.